### PR TITLE
Feature/latest narrative

### PR DESCRIPTION
**Pull request by brettminnie**
Partial solution for #121

### CODE REVIEW

File: src/TransformCore/PHE/HayApi/NarrativeBundle/Specification/AboutYou/AboutYouFeelingsAreAllZeroSpecification.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * @package TransformCore\PHE\HayApi\NarrativeBundle\Specification\AboutYou
+ */
+class AboutYouFeelingsAreAllZeroSpecification implements NarrativeSpecification
+{
+    /**
+     * @inheritDoc
+     */
+    public static function isSatisfiedBy(Questionnaire $questionnaire)
+    {
+        $normalisedFeelings = $questionnaire->getAboutYou()->getFeelings()->getNormalisedValues();
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/NarrativeBundle/Specification/AboutYou/AboutYouFeelingsAreAllZeroSpecification.php]: Multiple lines might be easier to read?

eg.
```php
$questionnaire
   ->getAboutYou()
   ->getFeelings()
   ->getNormalisedValues();
```

Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/NarrativeBundle/Specification/AboutYou/AboutYouFeelingsAreAllZeroSpecification.php]: Noted, i'll amend

Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/NarrativeBundle/Specification/AboutYou/AboutYouFeelingsAreAllZeroSpecification.php]: Nothing major, just a thought as I was going through it

**Comment by brettminnie** [comment]
:+1:

**Pull request merged**